apps/frontend/src/pages/[type]/[id]/settings/environment.vue

```diff
@@ -9,14 +9,15 @@ import {
 	UnsavedChangesPopup,
 	useSavable,
 } from '@modrinth/ui'
-import { injectApi } from '@modrinth/ui/src/providers/api.ts'
 import { defineMessages, useVIntl } from '@vintl/vintl'
 
+import { injectModrinthClient } from '~/providers/api-client.ts'
+
 const { formatMessage } = useVIntl()
 
 const { currentMember, projectV2, projectV3, refreshProject } = injectProjectPageContext()
 const { handleError } = injectNotificationManager()
-const api = injectApi()
+const client = injectModrinthClient()
```

Comment:
the client name feels a *little* confusing when used elsewhere like down below, makes me think localstorage or something about the frontend client. maybe in places like this I would destructure it to { labrinth } ? simplifies later calls and also makes it very explicitly clear which API you're calling, which is cool